### DISCUSSION TITLE
Condense GSA Bot Sub Package Notifs into One Post

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#253] [Suggestions]
Pretty much that. Purchases of sub-packages in collections create two posts in the respective discord channel for tracking activity. It would be nice if, instead of both posts, it'd be condensed into one post, with a slight addition to include the parent number and the collection name that the sub-pack was purchased from. 

<img width="332" alt="febdf55778672c9b9c369cc19185c16f" src="https://user-images.githubusercontent.com/44717221/182723362-b807abfe-39e3-4d17-9b84-50e06c26d2c5.png">

**Comment by mvaessen** [comment]
Hi, i've converted this into a suggestion

**Reply by Decollates** [reply]
Oops, sorry if it was submitted erroneously.

**Comment by mvaessen** [comment]
Closing due to overlap with #374
